pkg/kubecost/allocationprops.go

```diff
@@ -298,7 +298,7 @@ func (p *AllocationProperties) GenerateKey(aggregateBy []string, labelConfig *La
 			} else {
 				labelName := labelConfig.Sanitize(strings.TrimPrefix(agg, "label:"))
 				if labelValue, ok := labels[labelName]; ok {
-					names = append(names, fmt.Sprintf("%s=%s", labelName, labelValue))
+					names = append(names, fmt.Sprintf("%s", labelValue))
```

Comment:
Do we want to do this same thing for annotations?

Reply:
We've been treating labels and annotations as the "same" thing for a little while now, so I think the answer here is yes. @nikovacevic?

Reply:
Yeah, let's do it for annotations, too.